src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -71,6 +72,8 @@ export default class rd2EntryForm extends LightningElement {
 
     @track isElevateWidgetDisplayed = false;
     isElevateCustomer = false;
+    elevateAuthToken;
```

Comment:
Let's rename `elevateAuthToken` to `paymentMethodToken` as used by the Elevate payments services API.